docs/sphinx/user_guide/feature/list.rst

```diff
@@ -0,0 +1,257 @@
+
+.. _list-label:
+
+=====
+Lists
+=====
+
+The ``list`` type in the CAMP library is a powerful tool for type manipulation and 
```

Comment:
```suggestion
The ``list`` type in Camp is a powerful tool for type manipulation and 
```

Reply:
Already corrected in most recent version